### DISCUSSION TITLE
Tracability (notifications, logs, etc...) when token revoked

### DISCUSSION THREAD

**Discussion by yukihiko-shinoda** [community/community#21314] [Apps, API and Webhooks]
Doesn't Github notify or log when revoke token?

I have several norification mails when I had created Personal access tokens.
Although, there are no Personal access token list in my GitHub account.
I checked [Settings] -> [Developer settings] -> [Personal access token] .

I checked manuals: [Token expiration and revocation - GitHub Docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/token-expiration-and-revocation)
Maybe, my case corresponds to the case: [Token expired due to lack of use](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/token-expiration-and-revocation#token-expired-due-to-lack-of-use)
However, I can't check cleary reason because I can't find any tracability function like notification mails or logs.

FYI: In my case, the last creation time of Personal access token is 11-22-2020.

**Comment by airtower-luna** [comment]
So far I've always received mail notifications for tokens getting close to their expiration date, and when they expired. I haven't been in the situation that a token expired due to lack of use, so I don't have experience with that specifically.
